src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -119,6 +119,9 @@ export class OverflowMenuPane extends Dialog implements AfterViewInit {
 	}
 
 	protected listItems() {
-		return Array.from<any>(this.elementRef.nativeElement.querySelectorAll(".bx--overflow-menu-options__btn:not([disabled])"));
+		const list = [];
+		const selector = ".bx--overflow-menu-options__option:not([disabled])";
+		this.elementRef.nativeElement.querySelectorAll(selector).forEach(item => list.push(item.firstElementChild));
```

Comment:
What was wrong with `Array.from(this.elementRef.nativeElement.querySelectorAll(selector));`?

Reply:
the disabled option was still part of the disabled listItems so I had to do it in a different way

Reply:
There's nothing your doing here that isn't the exact same as doing `Array.from(this.elementRef.nativeElement.querySelectorAll(selector));` ... so I'm still not sure why this changed

Reply:
im getting the firstElementChild of each Node in the querySelectorAll thats why

Reply:
Hm, so really you want `const selector = ".bx--overflow-menu-options__option:not([disabled]) .bx--overflow-menu-options__btn"`